index.html

```diff
@@ -23,6 +23,11 @@
     <script src="../scripts/secrets.js"></script>
     <script src="../scripts/connector.js"></script>
     <script src="../scripts/handlers.js"></script>
+    <style>
+      *{
+        scroll-behavior: smooth;
+      }
+    </style>
```

Comment:
Let's create a new folder `styles` and add a `.css` file there. Let's not add inline CSS here.
Also, can you explain a bit more, why is this necessary?

Reply:
Ya... definitely it's a good practice to add a new .css file and write the code there. 
The scroll-behavior property in CSS allows us to define whether the scroll location of the browser jumps to a new location or smoothly animates the transition when a user clicks a link that targets an anchored position within a scrolling box.
For Example, In your Code, when you click:- Play with Notification link, you will scroll down smoothly instead of directly jumping to it.
You can learn more about it from:- https://www.w3schools.com/cssref/pr_scroll-behavior.asp
👍